### DISCUSSION TITLE
W3C Community Group Chair Nominations

### DISCUSSION THREAD

**Discussion by robertlong** [omigroup/omigroup#17] [General]
Nominate yourself or someone else for chairs for the Metaverse Interoperability W3C Community Group.

## What's involved with being a chair of a W3C Working Group?
These individuals will be responsible for making sure the group fulfills the [W3C's community group requirements](https://www.w3.org/community/about/process/#operational-agreements) as well as any of our group's operational agreements which we will also need to determine. Chairs will lead meetings, administer the GitHub repository, schedule meetings, send out agenda emails to the mailing list, and other administrative tasks.

## When will we be voting on chairs?
In the second meeting, we announced that we would hold an election on the 4th meeting. There's not a date set for that yet and we may push that date back should we not have enough candidates or diversity of applicants.

## How many chairs will there be?
Ideally we would only have 2-3 chairs. It could become somewhat challenging to manage more than a few chairs.

## How long is a chairs term? 
We believe that the first group of Chairs should serve for a 3 month term, at which time we will have another election. We believe we can increase term lengths as we grow. We believe this will help us become remain flexible, and diverse.

**Comment by TrevorFSmith** [comment]
I nominate @robertlong

**Reply by mrmetaverse** [reply]
++1 Robert

**Reply by Vytek** [reply]
++1 Robert

**Reply by arpu** [reply]
++1 Robert

**Reply by fire** [reply]
++1 Robert

**Comment by lalalune** [comment]
I nominate Jesse from AngellXR and Jin from Webaverse as both have shown a track record of commitment to the goal and ability to write things.

**Reply by mrmetaverse** [reply]
++1 Jin 
++1 I accept the nomination, and would be honored.

**Reply by ryota-mitarai** [reply]
++1 Jin

**Comment by gobengo** [comment]
I can help support chairs with some experience from w3c [SocialWG](https://www.w3.org/wiki/Socialwg) or pull in advice from people who supported that group.

**Comment by robertlong** [comment]
@dmitrizagidulin suggested in the W3C SocialCG call today that we might want to consider chair terms that increase in length as the community matures. Since OMI is such a young org - I agree with this suggestion. Many qualified people have yet to even hear about our group yet. If someone comes along in 3 months that would be perfect to help chair our group I think it'd be great if there was an election scheduled within a reasonable period of time to get them on board.

So perhaps we have our election within the next couple meetings? We elect 2-3 chairs and then in 3 months we do it again. After that one, we have another election in 6 months, then a year and then we can cap it at a year. Obviously these will be renewable terms and I would expect many people to be re-elected. 

We have some great people involved in this community and being proposed here. I'm not detracting from anyone's qualifications, but I think this group will change quickly over the next year or two and we will need the correct leadership to handle that.

Thoughts?

**Reply by mrmetaverse** [reply]
Brilliant! 

It's half baked, but the first thought that comes to mind is chairs should be encouraged to be mindful of associating "planned obsolescence" with efforts, as that can lead to some feeling their decisions "don't matter anyways," or that they can "defer that decision* until the next round of chairs." 

I think this is a fantastic mode that could boost our agility and our resilience.

**Comment by robertlong** [comment]
After our conversation in meeting 5, we'll be following a gradually increasing term duration for chairs. The first election tomorrow will have a term length of 3 months ending on the nearest weekly meeting (June 2nd - August 4th).

**Comment by mrmetaverse** [comment]
Before tomorrows vote, a few of us want to nominate Evo as a chair. <3
